sdk/tests/fixtures/certs/ed25519.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
I'd like to have a README at the root of this new certs directory explaining that these are throwaway keys. There will likely be some question about checking in private keys into a public GitHub repo.

Reply:
👍